### PR TITLE
Add github files

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Add github files in order to follow our standards and the good practice to work with opensource.  

## What
Add github files 

## Why
- Following the [opensource guide](https://opensource.guide/) recommendations and checklists for contributors choose a project to contribute with
- Following our standards
- Help the users/contributor track properly an issue and a PR

## How
Applying the same ones defined in: https://github.com/integr8ly/installation/tree/master/.github

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
```
Comment (camilamacedo86) [inline .github/ISSUE_TEMPLATE.md]: Hi @JameelB,

Is this env good for this project? Could you help me here by checking this PR?

Comment (pb82) [reply .github/ISSUE_TEMPLATE.md]: I don't think it's required here because the operator is usually running inside a container with a fixed base image.

Comment (camilamacedo86) [reply .github/ISSUE_TEMPLATE.md]: What env you do not think is required? @pb82

Comment (pb82) [reply .github/ISSUE_TEMPLATE.md]: Ah sorry, i meant the operating system. Not sure if we need that info in the typical usage scenarios but it also won't hurt.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
```
Comment (pb82) [inline .github/ISSUE_TEMPLATE.md]: Can we remove this? I don't think it makes sense for an operator.

Comment (camilamacedo86) [reply .github/ISSUE_TEMPLATE.md]: Hi @pb82,

Sure. However, could not the gitea-operator installed by ansible scripts? Also, note that the dev/user/contributor is able to remove everything that may not be required. The idea just helps/try to ask all required info.  

Please, let me know if still making sense remove the ansible line and if have anything else that you would like to remove here.

**Pull request merged**